### DISCUSSION TITLE
Can I have complex markdown formatting in the answer?

### DISCUSSION THREAD

**Discussion by kgoedert** [st3v3nmw/obsidian-spaced-repetition#758] [Q&A]
Hello,

I am using the multiline flashcards, and I can format the question part of the card pretty well. But when I try to format the answer part the same way, it loses all formatting. I would like to have some markdown in text and some code blocks too. Is it possible?

**Comment by jbuddha** [comment]
I have the same usecase. I want to use Callouts to provide hints for the answers.
